deploy/packaging/tr1d1um_spruce.yaml

```diff
@@ -123,6 +123,11 @@ log:
 # information using argus.
 webhook:
 
+  # requirePartnerIDs, if true, will require webhooks to have partnerIDs in the request
+  # for them to be registered.
+  # Defaults to 'false'.
+  requirePartnerIDs: false
```

Comment:
We should flip this, so requiring partner IDs is the default.

Reply:
flipped